.github/workflows/ci.yaml

```diff
@@ -0,0 +1,80 @@
+name: CI
```

Comment:
I'm not very familiar with github actions.

I just tried to create the config file based on https://github.com/FriendsOfSymfony/FOSRestBundle/blob/3.x/.github/workflows/continuous-integration.yml